content/id/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -984,14 +984,14 @@ sehingga Deployment tidak akan dapat dikembalikan.
 
 ## Deployment Canary
 
-Jika kamu ingin merilis ke sebagian pengguna atau server menggunakan Deployment, 
+Jika kamu ingin merilis ke sebagian pengguna atau server menggunakan Deployment,
 kamu dapat membuat beberapa Deployment, satu tiap rilis, dengan mengikuti pola canary yang didesripsikan pada
 [mengelola sumber daya](/id/docs/concepts/cluster-administration/manage-deployment/#deploy-dengan-canary).
 
 ## Menulis Spesifikasi Deployment
 
 Sebagaimana konfigurasi Kubernetes lainnya, Deployment memerlukan kolom `apiVersion`, `kind`, dan `metadata`.
-Untuk informasi umum tentang penggunaan berkas konfigurasi, lihat dokumen [deploy aplikasi](/id/docs/tutorials/stateless-application/run-stateless-application-deployment/),
+Untuk informasi umum tentang penggunaan berkas konfigurasi, lihat dokumen [deploy aplikasi](/docs/tutorials/stateless-application/run-stateless-application-deployment/),
 mengatur kontainer, dan [memakai kubectl untuk mengatur sumber daya](/id/docs/concepts/overview/working-with-objects/object-management/).
```

Comment:
if someone translates the run-stateless-application-deployment page, we need to update this line to add /id, right?
in this case, I think we need to check for every translated page, whether or not has impact to other pages
but it's only workaround while waiting for this issue https://github.com/kubernetes/website/issues/18403

Reply:
hi @irvifa why did you remove this changes?

I think removing /id/ from `deploy aplikasi` is fine.
I'll create a checker to accommodate the workaround on my previous comment :crossed_fingers: